packages/sanity/src/structure/panes/document/documentPanel/banners/OpenReleaseToEditBanner.tsx

```diff
@@ -0,0 +1,124 @@
+import {Flex, Text} from '@sanity/ui'
+import {useCallback, useMemo} from 'react'
+import {
+  getReleaseIdFromReleaseDocumentId,
+  getReleaseTone,
+  getVersionFromId,
+  isDraftId,
+  isPublishedId,
+  Translate,
+  useActiveReleases,
+  useDocumentVersions,
+  useDocumentVersionSortedList,
+  useSetPerspective,
+  useTranslation,
+  VersionInlineBadge,
+} from 'sanity'
+import {structureLocaleNamespace} from 'sanity/structure'
+
+import {Button} from '../../../../../ui-components'
+import {Banner} from './Banner'
+
+export function OpenReleaseToEditBanner({
+  documentId,
+  isPinnedDraftOrPublished,
+}: {
+  documentId: string
+  isPinnedDraftOrPublished: boolean
+}): React.JSX.Element | null {
+  const {onlyHasVersions} = useDocumentVersionSortedList({documentId})
+
+  /** this banner should only be rendered in specific cases
+   * 1. when the document is not a draft or published
+   * 2. when the document only has version or versions
+   * 3. when the global perspective is not set (published or draft)
+   */
+
+  if (isDraftId(documentId) || isPublishedId(documentId)) {
```

Comment:
Just wondering: can this be simplified to `!isVersionId(documentId)`?

```suggestion
  if (!isVersionId(documentId)) {
```

Reply:
Probably can, I will try it out and make sure that it works as intended

Reply:
Done!